src/coordinator/roles.js

```diff
@@ -161,6 +257,88 @@ class Roles {
       .get(`${this._baseUrl}/organizations/${organizationId}/roles`)
       .then((roles) => toCamelCase(roles));
   }
+
+  /**
+   * Remove an application from a role
+   *
+   * API Endpoint: '/applications/:applications_id/roles/:roleId'
+   * Method: DELETE
+   *
+   * @param {string} organizationId The ID of the organization
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {string} applicationId The UUID formatted ID of the application
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.roles.delete('4f0e51c6-728b-4892-9863-6d002e61204d');
```

Comment:
Needs updated to be `roles.removeApplication(....)` with the 3 args

Reply:
done